git/intermediate_git/a_deeper_look_at_git.md

```diff
@@ -151,9 +150,9 @@ You might be feeling overwhelmed at this point, so let's recap what we've learne
 
 The following questions are an opportunity to reflect on key topics in this lesson. If you can't answer a question, click on it to review the material, but keep in mind you are not expected to memorize or master this knowledge.
 
-- [Explain what it means for branches to be pointers.](https://git-scm.com/book/en/v2/Git-Branching-Branches-in-a-Nutshell)
-- [How can you amend your last commit?](https://git-scm.com/book/en/v2/Git-Basics-Undoing-Things)
-- [What are some different ways to rewrite history?](https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History)
+- [How can you amend your last commit?](https://www.theodinproject.com/lessons/javascript-a-deeper-look-at-git#changing-the-last-commit)
+- [What are some different ways to rewrite history?](https://www.theodinproject.com/lessons/javascript-a-deeper-look-at-git#changing-multiple-commits)
+- [What does it mean for branches to be pointers?](https://www.theodinproject.com/lessons/javascript-a-deeper-look-at-git#branches-are-pointers)
```

Comment:
```suggestion
- [How can you amend your last commit?](#changing-the-last-commit)
- [What are some different ways to rewrite history?](#changing-multiple-commits)
- [What does it mean for branches to be pointers?](#branches-are-pointers)
```
Because these links all reference ids on this page, the href for all of them can be simplified to just have the id rather than the full URL.

Reply:
Thanks for spotting that and for the suggestion - I've fixed it now :-)